### ISSUE TITLE
ATL compatibility for predefined CE effects in v9

### ISSUE THREAD

**Issue by tokamachina** [DFreds/dfreds-convenient-effects#93]
The provided conditions/effects from Convenient Effects are not working with Active Token Effects (Active Token Lighting.)
I think this is because the keys for lighting have changed.
From https://github.com/kandashi/Active-Token-Lighting/blob/main/src/updateManager.js 

```
static v9UpdateEffect(change) {
        switch (change.key) {
            case "ATL.preset":
            case "ATL.brightSight":
            case "ATL.dimSight":
            case "ATL.height":
            case "ATl.img":
            case "ATL.mirrorX":
            case "ATL.mirrorY":
            case "ATL.rotation":
            case "ATL.scale":
            case "ATL.width":
                break;
            case "ATL.dimLight":
                change.key = "ATL.light.dim";
                break;
            case "ATL.brightLight":
                change.key = "ATL.light.bright";
                break;
            case "ATL.lightAnimation":
                change.key = "ATL.light.animation";
                break;
            case "ATL.lightColor":
                change.key = "ATL.light.color";
                break;
            case "ATL.lightAlpha":
                change.key = "ATL.light.alpha";
                break;
            case "ATL.lightAngle":
                change.key = "ATL.light.angle"
                break;
        }
       return change
}
```
I assume changing the DF CE preset definitions should fix it.
(One thing that was tripping me up: this ATL change has also broken some of its compatibility with DAE - the auto-fill for the keys suggests the old keys instead of the new ones.)

**Comment by mateoayat** [comment]
I'm having the same issue

**Comment by DFreds** [comment]
There is a minor issue with ATE at the moment around light animations that Kandashi is going to fix. Once that is out and verified to work, this fix will go out in v2.0.1.

**Comment by DFreds** [comment]
Fixed in v1.15.0. Requires ATE 0.3.03
